docs/UserGuide.adoc

```diff
@@ -249,40 +245,32 @@ image::ug-findcExamplePart2.png[]
 image::ug-findcExamplePart3.png[]
 
 ==== Common errors / problems
-You might face some errors or difficulties when you `find-c`. In this section, you will be able to understand these errors and resolve them. You will also get a better understanding of the reply from the result box when using `find-c`.
+You might face some errors or difficulties when you `find-c`. In this section, you will be able to understand these errors and resolve them.
```

Comment:
I think we can afford to be more concise? How about "here are some common errors and problems:"